modules/dkimVerifier.jsm.js

```diff
@@ -134,6 +133,13 @@ const PREF_BRANCH = "extensions.dkim_verifier.";
  * @property {String[]} [errorStrParams]
  * @property {Boolean} [hideFail]
  * @property {Boolean} [keySecure]
+ * @property {number|null} [sigKeyLength]
+ * @property {number|null} [timestamp]
+ * @property {number|null} [expiration]
+ * @property {string} [sigAlgo]
+ * @property {string} [hashAlgo]
+ * @property {string[]} [signedHeaders]
+ * @property {String|undefined} [verifiedBy]
```

Comment:
Note that I will also add new values to this `dkimSigResultV2` structure, too as version 2.1. Which are slightly different from yours.
This could potentially cause smaller issues if results are saved with your version, and later the add-on is upgraded.

The issues are probably only information lose of the fields newly added by you.
Don't know what the best solution would be here (e.g. what should the version number be here?) and how much we even need to care about it,

But probably makes sense to at least name the once we both added the same.

Reply:
I've renamed 
- `sigAlgo` to `algorithmSignature`, 
- `hashAlgo` to `algorithmHash` and 
- `sigKeyLength` to `keyLength` 

properties to match yours in `master`.
I've an additional property `verifiedBy`, which holds the `authserv_id` for results from ARH records.
All other attributes match yours.

This should facilitate future backports. Since `authserv_id` is an optional property, I guess the same version doesn't hurt.